.github/workflows/upload-docs.yaml

```
@@ -0,0 +1,66 @@
+name: Upload Docs
+
+on:
+  pull_request:
```

```suggestion
  push:
```
rather than PR creation